lib/services/task.ts

```diff
@@ -1,96 +1,137 @@
-import { db, Task, USER_ID } from '../db';
-// import { prisma } from '../prisma';
-
 /** This file holds the Task service
  * The Task service is responsible for CRUD operations on the tasks table in the database
  */
 
+// TODO:
+// Update the types with Prisma types - add Promise<type> returns
+
 export class TaskService {
```

Comment:
I might get rid of the task service (and other services) in the future - they could be a bit redundant, and we can just build a fetch instance that's more user friendly instead of a whole service like this.